portal/src/main/webapp/js/src/study-view/component/PieChart.js

```diff
@@ -89,14 +89,17 @@ var PieChart = function(){
     var labelLT = 5; //label length threshold
     var labelHeaderLT = 4; //label header length threshold
 
+    //The attributes will be converted to table view after initialization
+    var tableAttrs = {'CANCER_TYPE':1, 'CANCER_TYPE_DETAILED':1};
+
     //This function is designed to draw Pie Labels based on current color the
     //Pie Chart has. Pagging function will be added when the number of labels
     //bigger than 5.
     function addPieLabels() {
         var _filters =[];
 
-        $('#' + DIV.mainDiv + ' .study-view-pie-label').html("");
-        $('#' + DIV.mainDiv + ' .study-view-pie-label-copy').html("");
+        d3.selectAll('#' + DIV.mainDiv).selectAll('.study-view-pie-label').selectAll('*').remove();
```

Comment:
I've changed in here: https://github.com/zhx828/cbioportal/commit/2f06bf7dac17a6b625550346f5b334e54e90f5ec